.github/workflows/on-schedule-nightly.yaml

```diff
@@ -0,0 +1,11 @@
+name: "On schedule: healthcheck of standard and latest setup"
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 2 * * *' # Every day at UTC 2.00
+
+jobs:
+  standard:
+    uses: ./.github/workflows/healthcheck-standard.yaml
+  latest:
+    uses: ./.github/workflows/healthcheck-latest.yaml
```

Comment:
Would be nice to be able to reuse the same workflow to avoid duplicate "code". Looks like the only difference between these two workflowss is the version for docker images?
Might be possible with use an optional input "USE_LATEST" to the workflows + some bash magic.

For example
```bash
if ${{ inputs.USE_LATEST }} // Not sure if this is valid syntax
then
  DMSS_TAG=latest JOB_API_TAG=latest docker-compose pull
else
  docker-compose pull
fi

```

Reply:
I thought about doing that, but there are more differences that that.
- dm-cli from pypi or github main, the latter needed both cloning and a different pip install command
- create-dm-app from npm or github main, the latter needing both cloning and a different run command
- dc env variables, as mentioned
- Update of @development-framework packages to latest

I could do as you propose, but I think it would be hard to read. I think it is better to have them separately, and then you can use a diff tool in your editor to see the differences and similarities